### ISSUE TITLE
LOG_LEVEL

### ISSUE THREAD

**Issue by aifrog** [B-urb/doclytics#120]
I trying to understand the behaviour of doclytics. I experimenting a bit and tried to change the LOG_Level. However, I am not successful yet. Would "LOG_LEVEL: TRACE" work in a docker environment? Would this result in a detailed log by document where I would see the returned json by document?

**Comment by b-urb** [comment]
Yes, it should work and print the full responses of the requests. What is your problem with the behavior ? Maybe I can help.
